repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeRoutingServlet.scala

```diff
@@ -236,7 +235,11 @@ class ReposeRoutingServlet @Inject()(@Value(ReposeSpringProperties.CORE.REPOSE_V
   }
 
   def preProxyMetrics(destination: Destination): Unit = {
-    reportingService.incrementRequestCount(destination.getId)
+    metricsService.foreach { metSer =>
+      metSer.getRegistry.meter(
+        MetricRegistry.name("org.openrepose.core.RequestDestination", destination.getId)
+      ).mark()
+    }
```

Comment:
This change moves where this metric can be found, so it should probably be called out in the upgrade notes.

Reply:
Added to the upgrade guide.